README.md

```diff
@@ -5,15 +5,19 @@ Targets small and large scale private clouds.
 
 # Install
 
-Get the LXD snap
+1. Get the LXD snap
```

Comment:
Why was this file changed? I think we should undo the changes here to keep the commit clean.

Reply:
I might need some assistance here :')

Reply:
Easiest way is probably just copy the same file content on the master branch on github and paste it into your local file.